Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 public struct RouteResponse {
     public var code: String?
     public var message: String?
-    public var error: String?
+    public var error: DirectionsError?
     public let uuid: String?
```

Comment:
This ID isn’t guaranteed to be a valid UUID. (The Swift standard library’s UUID type expects UUIDs to conform to RFC 4122 version 4.) So we should call it `identifier` instead.